packages/ui/src/views/pull-request/pull-request-list-header-title.tsx

```diff
@@ -0,0 +1,60 @@
+import { cn } from '@utils/cn'
+
+import { Icon } from '../../components/icon'
+import { Text } from '../../components/text'
+
+export const PullRequestListHeaderTitle = ({
+  setHeaderFilter,
+  headerFilter,
+  closed_prs,
+  open_prs
+}: {
+  setHeaderFilter: (state: string) => void
```

Comment:
As a general rule, it's a good idea to pass very focused event callbacks instead of functions where appropriate. So in this case, instead of passing a setter function, pass 2 event style functions. This puts the onus of what happens in the control of the caller.
```suggestion
  onOpenClick: () => void
  onCloseClick: () => void
```

Reply:
done